sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -85,7 +85,7 @@ internal static DetectLanguageResultCollection ConvertToDetectLanguageResultColl
             var detectedLanguages = new List<DetectLanguageResult>(results.Documents.Count);
 
             //Read errors
-            foreach (DocumentError error in results.Errors)
+            foreach (InputError error in results.Errors)
```

Comment:
None of these renamed types are public, right?

Reply:
Correct. Also, to answer your other question, this is an example of something that broke after re-generating (fortunately it's just an internal type so it's not an actual breaking change for consumers of the library).